package.json

```diff
@@ -14,7 +14,7 @@
     "test:unit-watch": "jest --watch",
     "test:coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
     "start": "webpack-dev-server --open --config webpack.dev.js",
-    "clean": "rm -rf dist",
+    "clean": "rimraf dist",
```

Comment:
@arnabk so rimraf is back! Turns out in this script travis doesn't use it anymore.